storey/drivers.py

```diff
@@ -365,16 +365,16 @@ async def _load_aggregates_by_key(self, container, table_path, key):
         else:
             raise V3ioError(f'Failed to get item. Response status code was {response.status_code}: {response.body}')
 
-    async def _load_by_key(self, container, table_path, key):
+    async def _load_by_key(self, container, table_path, key, attributes):
         self._lazy_init()
 
-        response = await self._v3io_client.kv.get(container, table_path, key, raise_for_status=v3io.aio.dataplane.RaiseForStatus.never)
+        response = await self._get_item(container, table_path, key, attributes)
         if response.status_code == 404:
             return None
         elif response.status_code == 200:
             res = {}
             for name in response.output.item.keys():
-                if not name.startswith(self._aggregation_attribute_prefix):
+                if not name.startswith((self._aggregation_attribute_prefix, self._aggregation_time_attribute_prefix)):
```

Comment:
❓

Reply:
There are 2 different system prefixes, and im ignoring both here

Reply:
Huh, I didn't know you could pass a tuple there.